app/src/test/java/org/simple/clinic/drugs/EditMedicineUpdateTest.kt

```diff
@@ -31,4 +43,48 @@ class EditMedicineUpdateTest {
             hasEffects(RefillMedicines(patientUuid)),
         ))
   }
+
+  @Test
+  fun `when filled prescribed drugs are fetched and some of the drugs are updated on the same day, then show set edit medicine button state to save button`() {
```

Comment:
Since this is business logic, the name of the test should be as close to domain specific terms instead of implementation specific terms.

Suggestion: "when the prescription has been updated on the current date, then the save medicine button must be shown".